### ISSUE TITLE
Deactivate embargo button is missing context

### ISSUE THREAD

**Issue by rjkati** [samvera/hyrax#6782]
### Descriptive summary

On dev.nurax.samvera.org (Hyrax 5.0.1), the `Deactivate Embargo` buttons on the `Expired Active Embargoes` tab are in table cells which are missing context and do not meet WCAG AA requirements

### Steps to reproduce the behavior in User Interface (UI)

1. In Chrome, download the Site Improve browser extension
Go to https://dev.nurax.samvera.org/embargoes?locale=en#expired and run the Site Improve browser extension

![expired_active_missing_context](https://github.com/samvera/hyrax/assets/32644254/74435cb8-1238-425a-bc96-84aae79d4553)

**Comment by christhepianist** [comment]
Tested and confirmed.

**Comment by christhepianist** [comment]
Tested and confirmed that this is no longer an issue.
